src/main/java/net/minecraftforge/common/util/TransformationHelper.java

```diff
@@ -108,9 +110,9 @@ public static boolean epsilonEquals(Vector4f v1, Vector4f v2, float epsilon)
 
     public static class Deserializer implements JsonDeserializer<Transformation>
     {
-        private static final Vector3f ORIGIN_CORNER = new Vector3f();
-        private static final Vector3f ORIGIN_OPPOSING_CORNER = new Vector3f(1f, 1f, 1f);
-        private static final Vector3f ORIGIN_CENTER = new Vector3f(.5f, .5f, .5f);
+        private static final Vector3f ORIGIN_CORNER = Origin.Corner.vec();
+        private static final Vector3f ORIGIN_OPPOSING_CORNER = Origin.OpposingCorner.vec();
+        private static final Vector3f ORIGIN_CENTER = Origin.Center.vec();
```

Comment:
It looks like we can remove these constants (which should be of no effect to binary compatibility, since these are `private`) and simply inline the three remaining uses of these constants.